FirebaseRemoteConfigSwift/Sources/FirebaseRemoteConfigValueDecoderHelper.swift

```diff
@@ -47,4 +52,18 @@ struct FirebaseRemoteConfigValueDecoderHelper: FirebaseRemoteConfigValueDecoding
     }
```

Comment:
Should we consider throwing an error here to make behavior more consistent with the API you have added? 

If so, we could create an issue to track this for Firebase 10

Reply:
Maybe? Returning nil is the historical behavior for failing to extract json. The array and dictionary methods are new and don't have historical precedent.

We might want to wait to we do a more fundamental refactor of RemoteConfigValue.

Reply:
^ would be interested in discussing this a bit more! feels like `arrayValue` and `dictionaryValue` would be preferred over `jsonValue`

Reply:
Great point! There's no reason for `jsonValue` any longer and I replaced its usage with `dictionaryValue`.